chaoslib/openebs/jiva_controller_pod_failure.yaml

```diff
@@ -0,0 +1,54 @@
+- name: Derive PV from application PVC 
+  shell: >
+    kubectl get pvc {{ app_pvc }}
+    -o custom-columns=:spec.volumeName -n {{ app_ns }}
+    --no-headers
+  args:
+    executable: /bin/bash
+  register: pv
+
+- name: Record the jiva controller deployment of the PV
+  set_fact: 
+    # Depends on the naming convention in maya-apiserver (<pv-id>-rep)  
+    jiva_controller_deploy: "{{ pv.stdout }}-ctrl"
+
+- name: Get the resourceVersion of the target deploy before fault injection
+  shell: >
+    kubectl get deploy {{ jiva_controller_deploy }} -n {{ app_ns }} 
+    -o yaml | grep resourceVersion | awk '{print $2}' | sed 's|"||g'
+  args:
+    executable: /bin/bash
+  register: rv_bef
+
+- name: Get jiva controller pod belonging to the PV
+  shell: > 
+    kubectl get pods -l openebs.io/controller=jiva-controller
+    -n {{ app_ns }} --no-headers | grep {{ pv.stdout }} 
+    | shuf -n1 | awk '{print $1}'
```

Comment:
Can we derive the pod name from specs

Reply:
Thank you @gprasath Have pushed the changes